GVFS/GVFS.Installer/GVFS.Installer.Mac.csproj

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\GVFS.Build\GVFS.cs.props" />
+  <PropertyGroup>
+    <AssemblyName>GVFS.Installer.Mac</AssemblyName>
+    <RootNamespace>GVFS.Installer.Mac</RootNamespace>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <Platforms>x64</Platforms>
+    <RunTimeIdentifiers>osx-x64</RunTimeIdentifiers>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+  </PropertyGroup>
+    
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
+    <Version>$(GVFSVersion)</Version>
+    <NoWarn>2008</NoWarn>

```

Comment:
What does this line mean? I don't see it in our other projects.

Reply:
This is to get the following warning ignored
```
  GVFS.Mac -> /Users/Ameen/Work/GVFS/BuildOutput/GVFS.Mac/bin/x64/Debug/netcoreapp2.1/osx-x64/gvfs.dll
CSC : warning CS2008: No source files specified. [/Users/Ameen/Work/GVFS/VFSForGit/GVFS/GVFS.Installer.Mac/GVFS.Installer.Mac.csproj]
  GVFS.Installer.Mac -> /Users/Ameen/Work/GVFS/BuildOutput/GVFS.Installer.Mac/bin/x64/Debug/netcoreapp2.1/osx-x64/GVFS.Installer.Mac.dll
  GVFS.Tests -> /Users/Ameen/Work/GVFS/BuildOutput/GVFS.Tests/bin/x64/Debug/netcoreapp2.1/GVFS.Tests.dll
  GVFS.FunctionalTests -> /Users/Ameen/Work/GVFS/BuildOutput/GVFS.FunctionalTests/bin/x64/Debug/netcoreapp2.1/osx-x64/GVFS.FunctionalTests.dll
  GVFS.UnitTests -> /Users/Ameen/Work/GVFS/BuildOutput/GVFS.UnitTests/bin/x64/Debug/netcoreapp2.1/osx-x64/GVFS.UnitTests.dll

Build succeeded.

CSC : warning CS2008: No source files specified. [/Users/Ameen/Work/GVFS/VFSForGit/GVFS/GVFS.Installer.Mac/GVFS.Installer.Mac.csproj]
    1 Warning(s)
    0 Error(s)
```

Reply:
This will also prevent the creation of unnecessary build outputs, so it seems like the right thing to do. Take a look at the prebuild csproj as an example.

Reply:
Agree. Was able to [figure out](https://stackoverflow.com/questions/47179705/net-core-override-default-build-targets) why my build target overrides were not working.

Now overriding Build, Rebuild and Publish.